### PR TITLE
[68] Implement get applied jobs API @GET

### PR DESCRIPTION
**Pull request by SuitedIceZ**
## What did you do
- implement ```GET /jobs/applied``` endpoint for actor to get applied jobs and filtering
- only able to filter title, jobStatus and applicationStatus
- title is sub-string search, insensitive case
- if not send any parameters (according to design)
  - jobStatus will be ['PENDING']
  - applicationStatus will be ['OPEN','SELECTING',''SELECTION_END']
  - title will be undefined (search any title)

## Demo
- https://api-dev.modela.miello.dev/swagger#/auth/AuthController_login
- login with (or any actor)
```
{
  "email": "actor6@gmail.com",
  "password": "password"
}
```
- https://api-dev.modela.miello.dev/swagger#/jobs/JobController_findAllApplied
- try get jobs with any jobStatus and applicationStatus combination
- should see all jobs and applicationStatus of the actor

## Limitation
- not have unit test yet

### CODE REVIEW

**Review by miello — COMMENTED** [review]
LGTM, please wait for review from @palmcm

File: apps/api/src/modules/job/job.repository.ts
Comment (palmcm) [inline apps/api/src/modules/job/job.repository.ts:178]: This will take latest application not the actor's application
```suggestion
        Application: {
          where: {
            actorId: userId,
          },
        },
```

**Review by palmcm — CHANGES_REQUESTED** [review]

File: apps/api/src/modules/job/job.repository.ts
Comment (palmcm) [inline apps/api/src/modules/job/job.repository.ts:181]: You can remove this, the database will be unique now

Comment (SuitedIceZ) [reply apps/api/src/modules/job/job.repository.ts:181]: ok

**Review by palmcm — APPROVED** [review]

**Pull request merged**
